packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -142,6 +142,11 @@ export interface ShimmerTokens extends IBackgroundColorTokens {
    * @defaultValue '100%' of the 'width' property
    */
   shimmerWaveWidth?: number | string;
+
+  /**
+   * overrides the useNativeDriver prop on the component's animation
+   */
+  useNativeDriver?: boolean;
```

Comment:
can we not expose it as a prop if we do the RN version check inside the component?

Reply:
I added the prop in case, say, a user is using some other fork of react-native or managed to patch react-native to include that fix, so they could override our default value.

Reply:
I feel like we are trying to cover all use cases rather than take it one step at a time. I personally think exposing this as a prop can be risky as it could affect performance of the control.